types/src/front/lib/actions/registry.ts

```diff
@@ -1,6 +1,15 @@
 import { DustAppType } from "../../../front/lib/dust_api";
+import { EnvironmentConfig } from "../../../shared/utils/config";
 
-const PRODUCTION_DUST_APPS_WORKSPACE_ID = "78bda07b39";
+export const PRODUCTION_DUST_WORKSPACE_ID = "0ec9852c2f";
+export const PRODUCTION_DUST_APPS_WORKSPACE_ID = "78bda07b39";
+
+const DUST_APPS_WORKSPACE_ID =
```

Comment:
I'm definitely not a fan to have types interfering with environment variables.

Reply:
Yes let's refactor this way:

-> we move the logic of switching workspaces in front
-> we parametrize the registry function with this value

Reply:
I wonder why the registry is in types BTW

Reply:
Would it be ok to move all this to front ? It not used anywhere else, I don't see neither why it's in types

Reply:
I'd say more than okay, it's purely front related.